lib/ecto/adapters/mysql.ex

```diff
@@ -30,6 +30,7 @@ defmodule Ecto.Adapters.MySQL do
     * `:port` - Server port (default: 3306)
     * `:username` - Username
     * `:password` - User password
+    * `:protocol` - Protocol used for the mysql client connection (default: tcp)
```

Comment:
We should clarify that this is only used for the ecto.create and ecto.drop commands. :)

Reply:
Correct me if I'm wrong, but it's only used for `ecto.load` and `ecto.dump` AFAICS?

Reply:
Yes, sorry. :)

Reply:
Cool, just pushed up the clarification. :)